lib/src/bluetooth_default_uuids.dart

```diff
@@ -2,6 +2,8 @@
 /// scripts/generate_default_uuids.dart. If you need more uuids please change
 /// the csv files and regenerate the file.
 
+// ignore_for_file: constant_identifier_names
```

Comment:
We could rewrite the generation script to use the preferred naming scheme for the final values in this file.

Though some people may not enjoy it since they will need to rename all their stuff. On the other hand the README says that this library implements a draft feature so how much can they really expect?

Reply:
Correct. You could add a transition path by generating:
````dart
  static const genericAttribute = BluetoothDefaultServiceUUIDS._(
      'Generic Attribute', '1801', '00001801-0000-1000-8000-00805f9b34fb', 0);
  @Deprecated('Use genericAttribute')
  static const GENERIC_ATTRIBUTE = genericAttribute;
````

and generate some dart doc and upgrade version. I don't have plan to tackle that though since that could also be a breaking change. I'll leave this issue up to you if that's ok.

Reply:
That's ok

Reply:
But I agree that it should be changed!